release.nix

```diff
@@ -68,8 +68,6 @@
 let
   overlay = pkgsNew: pkgsOld: {
 
-    grpc = pkgsNew.callPackage ./nix/grpc.nix { };
```

Comment:
I think we pinned `grpc` to ensure we got a supported C-core version. Though, I don't think it's bad to rely on the ambient `nixpkgs` version of `grpc` which, if the build breaks due to a nixpkgs pin upgrade, then it would stimulate us to maintain it.

However, if we delete this, we should also delete the Nix code.

Reply:
It looks like the `grpc` we instantiate mirrors to the one provided by `nixpkgs`, except the `nixpkgs` package has additional fixes that aren't included in `nix/grpc.nix`. I can hold off on this change for a separate PR though.